### PR TITLE
Refactor/674

### PR DESCRIPTION
**Pull request by 100vbaSasha**
Refactored the first part of the admin panel (Tabs, back buttons, save buttons, styles, etc)
### Checklist
- [ ] 🔽 My branch is up-to-date with "development" branch
- [ ] ✅ All tests passed locally
- [ ] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [ ] 🔗 Link pull request to issue

### CODE REVIEW

**Comment by AndriyLymych** [comment]
please, fix codesmells 
image

File: src/components/forms/category-form/category-form.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ const CategoryForm = ({ category, id, edit }) => {
           className={styles.saveCategoryButton}
           data-cy='save'
           type='submit'
-          title={SAVE_TITLE}
+          title='Зберегти'
```
Comment (AndriyLymych) [inline src/components/forms/category-form/category-form.js:170]: please, don't use magic string

File: src/components/forms/header-form/header-form.js [hunk 2]
```diff
@@ -100,6 +100,7 @@ const HeaderForm = ({ header, id }) => {
             value={tabsValue}
             onChange={handleTabsChange}
             aria-label='simple tabs example'
+            variant='fullWidth'
```
Comment (AndriyLymych) [inline src/components/forms/header-form/header-form.js:103]: same

File: src/components/forms/header-form/header-form.js [hunk 3]
```diff
@@ -129,7 +130,7 @@ const HeaderForm = ({ header, id }) => {
           className={styles.saveButton}
           data-cy='save'
           type='submit'
-          title={config.buttonTitles.HEADER_SAVE_TITLE}
+          title='Зберегти'
```
Comment (AndriyLymych) [inline src/components/forms/header-form/header-form.js:133]: same

File: src/components/forms/material-form/material-form.js [hunk 4]
```diff
@@ -240,6 +240,7 @@ function MaterialForm({ material, id }) {
                 value={tabsValue}
                 onChange={handleTabsChange}
                 aria-label='tabs'
+                variant='fullWidth'
```
Comment (AndriyLymych) [inline src/components/forms/material-form/material-form.js:243]: same

File: src/components/forms/material-form/material-form.js [hunk 5]
```diff
@@ -254,7 +255,7 @@ function MaterialForm({ material, id }) {
               className={styles.saveButton}
               data-cy='save'
               type='submit'
-              title={config.buttonTitles.SAVE_MATERIAL}
+              title='Зберегти'
```
Comment (AndriyLymych) [inline src/components/forms/material-form/material-form.js:258]: same

File: src/components/forms/size-form/size-form.js [hunk 6]
```diff
@@ -199,7 +199,7 @@ function SizeForm({ id, size }) {
             className={styles.saveButton}
             data-cy={materialUiConstants.save}
             type={materialUiConstants.types.submit}
-            title={config.buttonTitles.SAVE_SIZE_TITLE}
+            title='Зберегти'
```
Comment (AndriyLymych) [inline src/components/forms/size-form/size-form.js:202]: same

File: src/pages/order-item/order-item.js [hunk 7]
```diff
@@ -107,17 +107,19 @@ const OrderItem = ({ id }) => {
...
-        disabled={!dirty || !isValid}
-      >
-        {SAVE_ORDER}
-      </Button>
       <div className={classes.controlsBlock}>
         <BackButton />
+        <SaveButton
+          className={classes.saveBtn}
+          type='submit'
+          title='Зберегти'
```
Comment (AndriyLymych) [inline src/pages/order-item/order-item.js:115]: same

**Review by IlyaBelik — APPROVED** [review]

**Review by dmytro07 — DISMISSED** [review]

**Review by fulljsdev — DISMISSED** [review]

**Review by IlyaBelik — APPROVED** [review]

**Review by dmytro07 — APPROVED** [review]

**Review by fulljsdev — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Pull request merged**